js/modules/k6/k6.go

```diff
@@ -116,6 +116,26 @@ func (*K6) Group(ctx context.Context, name string, fn goja.Callable) (goja.Value
 	return ret, err
 }
 
+// AbortTest exposes abortTest function in the k6 module. When called ti will
+// interrupt the active goja runtime passed with ctx.
+func (*K6) AbortTest(ctx context.Context, extra ...goja.Value) {
+	state := lib.GetState(ctx)
+	rt := common.GetRuntime(ctx)
+	if state == nil {
+		rt.Interrupt(&common.InterruptError{Reason: common.AbortTestInitContext})
+		return
+	}
```

Comment:
as mentioned above, I think you should just drop this lines

Reply:
done eaa9ea97